backstopConfig.js

```diff
@@ -47,8 +47,8 @@ const config = {
     },
     {
       ...basic,
-      label: 'Link with class_is-active',
-      selectors: ['a.is-active'],
+      label: 'Link with class_nav__item',
+      selectors: ['a.nav__item'],
```

Comment:
Are you sure you should change something here? think after those changes you have problem with active link :)

Reply:
I did replacement everywhere by mistake. :)